<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConcretarOfrecido</class>
 <widget class="QDialog" name="ConcretarOfrecido">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="11" column="1">
    <widget class="QLineEdit" name="precio_concretado">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QDateEdit" name="fecha_conc_compra">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>110</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QDateEdit" name="fecha_pago">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>110</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_11">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Fecha de pago:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QComboBox" name="localizacion">
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_9">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Lugar de carga:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_15">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Observaciones:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_13">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Precio:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="tipo_negocio">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>35</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_12">
     <property name="maximumSize">
      <size>
       <width>190</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Fecha concreción compra:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tipo grano:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Kilos:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="moneda">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <item>
      <property name="text">
       <string>Peso</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dolar</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tipo de operación:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDateEdit" name="fecha_venta">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>110</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cliente">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="grano">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="kilos">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Cliente:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Precio Base (por tonelada):</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Fecha compra:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tipo moneda:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="precio_base">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_14">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Kilos:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLineEdit" name="kilos_concretados">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>65</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="13" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="13" column="0">
    <widget class="QLineEdit" name="id">
     <property name="maximumSize">
      <size>
       <width>1</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2" alignment="Qt::AlignRight">
    <widget class="QPushButton" name="pushButton_3">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Nuevo cliente</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QTextEdit" name="observaciones">
     <property name="maximumSize">
      <size>
       <width>99999</width>
       <height>70</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cliente</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>grano</tabstop>
  <tabstop>kilos</tabstop>
  <tabstop>precio_base</tabstop>
  <tabstop>moneda</tabstop>
  <tabstop>fecha_venta</tabstop>
  <tabstop>tipo_negocio</tabstop>
  <tabstop>observaciones</tabstop>
  <tabstop>localizacion</tabstop>
  <tabstop>fecha_pago</tabstop>
  <tabstop>fecha_conc_compra</tabstop>
  <tabstop>precio_concretado</tabstop>
  <tabstop>kilos_concretados</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>id</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
